synapse/storage/databases/main/keys.py

```diff
@@ -188,7 +188,7 @@ async def store_server_keys_json(
         # invalidate takes a tuple corresponding to the params of
         # _get_server_keys_json. _get_server_keys_json only takes one
         # param, which is itself the 2-tuple (server_name, key_id).
-        self._get_server_keys_json.invalidate((((server_name, key_id),)))
+        self._get_server_keys_json.invalidate(((server_name, key_id),))
```

Comment:
I think this is safe: before we have `invalidate( (t1) )` where `t1 = (t2,)`. We are removing the inner brackets around `t1`.

Reply:
Yeah, we have 4 sets of parens, if you expand it out you have:

1. Function call
2. Grouping (unneeded)
3. Tuple
4. Tuple

```python
invalidate( #function call
    ( # grouping
        ( # tuple
            ( # tuple
                server_name, key_id
            ),
        )
    )
)
```